docs/source/examples/advanced_projects.rst

```diff
@@ -0,0 +1,13 @@
+Advanced Projects
+=================================
+
+.. manim:: Projects1
+    :quality: medium
+    :save_last_frame:
+
+    class Projects1(Scene):
+        def construct(self):
+            t = TextMobject("Some advanced project")
+            self.add(t)
+            self.wait(1)
+
```

Comment:
I don't really see a point in adding these kind of placeholder projects; I'd rather see a brief statement explaining what it is that will be collected on this page.

```suggestion
This page is currently under construction. It will feature a selection of advanced projects built with manim.
```

Reply:
It is only for consistency: 
As I made placeholders everywhere, I thought this might also want one.
Also, as soon as some real projects are added here, you already have the template ;)
But as this is Work in Progress and will change very soon, I think its not worth spending to much time on that.